[JA] Fix typo Beatmapping/ja

lolita-is-godddd opened the pull request:
modify sentence, fix typo

---

sr229 commented:
Hey @lolita-is-godddd, next time when you make a one-commit PR, make sure you change the commit message. :wink:

lolita-is-godddd commented:
ah sry and I will rell you
the first change, moved "または" (means "or") to between two things for fix japanese syntax.
second, deleted "も" ("is also") for syntax.
third, changed Rankable to "Ranked可能" ("can be ranked").
forth, changed "ビートスペーシング" to "ビートのすき間" for natural translation.
fifth, deleted unnessary "r", fixing typo.

cl8n commented on wiki/Beatmapping/ja.md:
```diff
@@ -187,7 +187,7 @@ A couple of rules of thumb are:-
 ---------------------------
 
-<img src="Beatspacing.jpg" title="fig:A quick illustration" alt="A quick illustration" width="200" /> 直感的なプレイのために譜面はよい"ビートスペーシング"を持つべきです。一般的にタイムライン上で互いに近いBeatは、配置上でも近くなるべきで、逆も同様です。これは調和した譜面によって曲の流れを保証し、譜面をより楽しくさせることに繋がります。
+<img src="Beatspacing.jpg" title="fig:A quick illustration" alt="A quick illustration" width="200" /> 直感的なプレイのために譜面はよいビートのすき間を持つべきです。一般的にタイムライン上で互いに近いBeatは、配置上でも近くなるべきで、逆も同様です。これは調和した譜面によって曲の流れを保証し、譜面をより楽しくさせることに繋がります。
```
すき間 -> 隙間 ?

lolita-is-godddd replied:
fixed in `0a8f946`, check it out please?
